### ISSUE TITLE
Get current path of Colab

### ISSUE THREAD

**Issue by TimotheeWrightFicha** [googlecolab/colabtools#4069]
**Is your feature request related to a problem? Please describe.**

I just want an option to get the path of the colab i'm working on. Ex: My colab is in "/content/drive/Shared\ drives/Tech/AI/" 
Would it be that difficult to have a method in colab to get the path to the current directory ?

**Describe alternatives you've considered**

Only found solution is writing manually the path to the colab and cd but this is such a pain.

**Comment by EvanWiederspan** [comment]
To clarify, the path you provided looks like a path to a mounted Drive folder. Do you mean you're editing a Colab notebook that is also mounted on your VM, and you want that file path?

**Comment by TimotheeWrightFicha** [comment]
I actually have a colab in a shared drive (in which i have edit rights).

I want to get the path of this colab file, in the code of the colab itself.

So yes I mounted the drive because I thought that was the way to get access to the folder architecture. I hope it's clearer now :)

**Comment by cperry-goog** [comment]
got it, thanks for the request.

**Comment by g-i-o-r-g-i-o** [comment]
exactly, there's no way in hell to get the directory of the current notebook!

each time I have to write the complete path, incredible

**Comment by matteosox** [comment]
Hi, new to Colab! Is this not currently possible? If not, is there a roadmap in general, or a timeline in specific for this feature?

**Comment by matteosox** [comment]
Hi @cperry-goog , @EvanWiederspan , maybe also @colaboratory-team : any update on this? My company pays for GSuite, we use CoLab along with GCP. I'd love to start using the tool correctly, but not sure how to resolve some issues like this.

**Comment by g-i-o-r-g-i-o** [comment]
Any update to this? All my colab notebooks that have an harcoded path.

Each time that I move them, I need to modify the path. It's super annoying.

Furthermore having a space in the path "Colab Notebooks", leads to all kind of errors... There are some ancient tool that can't handle spaces.
CURRENT_DIR = '/content/drive/MyDrive/Colab Notebooks/Ultralytics'

**Comment by yuttai** [comment]
https://stackoverflow.com/questions/71080978/how-to-obtain-the-current-path-to-my-google-colab-file
The stack overflow work around is to hard-code the file name and global search it... Not a good approach either. 

Besides, the same reason results that I could not run a Colab Notebook locally without modifying it...

**Comment by g-i-o-r-g-i-o** [comment]
It's crazy that we don't have a fix to this. Every time I move a colab file, I need to check the paths.
